src/lib/states.ts

```diff
@@ -126,6 +126,21 @@ export class InverterStates {
                 state: {id: 'dailyEnergyYield', name: 'Daily energy yield', type: 'number', unit: 'kWh', role: 'value.energy'},
                 register: {reg: 32114, type: ModbusDatatype.uint32, length: 2, gain: 100}
             },
+            {
+                interval: UpdateIntervalID.LOW,
+                state: {id: 'generatedenergytoday', name: 'Daily generated Energy', type: 'number', unit: 'kWh', role: 'value.energy.produced'},
+                register: {reg: 32114, type: ModbusDatatype.uint32, length: 2, gain: 100},
+            },
+            {
+                interval: UpdateIntervalID.LOW,
+                state: {id: 'generatedenergymonth', name: 'Monthly generated Energy', type: 'number', unit: 'kWh', role: 'value.energy.produced'},
+                register: {reg: 32116, type: ModbusDatatype.uint32, length: 2, gain: 100},
```

Comment:
Hmm, i did not have these registers in my ModbusInterfaceDefinition document.
@Basti-RX Where do you got it from?
Can you send me the link to the doc?

Reply:
https://community.symcon.de/uploads/short-url/pqZXWOienoBK2AsEzGD2oH1bcPR.pdf

Reply:
Does it help you? Which Version you got?